src/Digdir.Library.Entity.EntityFrameworkCore/Features/Aggregate/AggregateExtensions.cs

```diff
@@ -44,11 +44,19 @@ internal static async Task HandleAggregateEntities(this ChangeTracker changeTrac
                 restored.OnRestore(aggregateNode, utcNow);
             }
 
-            if (aggregateNode.Entity is IUpdateableEntity updatable)
+            if (aggregateNode.Entity is IUpdateableEntity updatable && aggregateNode.State is AggregateNodeState.Modified)
```

Comment:
Tror jeg liker løsningen! Kunne du dog slått sammen ifene som rører IupdatableEntity? Gjerne lag to private metoder som sier nøyaktig hva du tester for 

IsModified || IsAddedWithDefaultCreatedAt

Reply:
Hmm, mente du private method på AggregateExtensions? 
Eller public halper method på aggregateNode? 🤔 

Sjekk ny kode